appserver/node-express/lib/db-client/qnaDoc/index.js

```diff
@@ -22,6 +22,18 @@ funcs.search = function (spec) {
   ).result();
 };
 
+/**
+ * Get one or more documents from the Samplestack database based ID(s).
+ * For single documents, apply a transform to include voting and
+ * reputation information.
+ *
+ * @param  {String} txid The transaction ID.
+ * @param  {Object|Array} spec An object with a document ID property. Example:
+ *   {"id":"b6ff5e47-2d08-4c20-9a16-4998fbb1bc10"}
+ * For multiple documents, an array of objects (TODO Is this correct? Where is
```

Comment:
The spec param can specify a single document or multiple (as coded). Where is an example of that method handling multiple documents? I don't see any examples. In such a case, is spec an array of objects (similar to the object for the singular case)?